README.md

```diff
@@ -27,33 +26,21 @@
 
 # What is Nx?
 
-🔎 **Nx is a set of Angular CLI power-ups for modern development.**
+🔎 **Extensible Dev Tools for Monorepos.**
 
 ## Nx Helps You
 
 ### Use Modern Tools
 
-Using Nx, you can add Cypress, Jest, Prettier, and Nest into your dev workflow. Nx sets up these tools and allows you to use them seamlessly. Nx fully integrates with the other modern tools you already use and love.
+Using Nx, you can add TypeScript, Cypress, Jest, Prettier, and Nest into your dev workflow. Nx sets up these tools and allows you to use them seamlessly. Nx fully integrates with the other modern tools you already use and love.
 
 ### Build Full-Stack Applications
 
-With Nx, you can build full-stack applications using Angular and Node.js frameworks such as Nest and Express. You can share code between the frontend and the backend. And you can use the familiar `ng build/test/serve` commands to power whole dev experience.
+With Nx, you can build full-stack applications using modern frameworks. You can share code between the frontend and the backend. And you can use the same `build/test/serve` commands to power whole dev experience.
 
-### Develop Like Google
+### Build like Google, Facebook, and Microsoft
```

Comment:
Build > Develop?

It's also the same verb as the previous point

Reply:
fixed